Spanner/tests/Unit/Batch/BatchSnapshotTest.php

```diff
@@ -95,7 +95,8 @@ public function testPartitionRead()
         $opts = [
             'index' => 'foo',
             'maxPartitions' => 10,
-            'partitionSizeBytes' => 1
+            'partitionSizeBytes' => 1,
+            'dataBoostEnabled' => true
```

Comment:
refactor the existing test to use / not-use the optional args provided by a data provider.